### PR TITLE
ref: Update dependencies

### PR DESCRIPTION
**Pull request by Swatinem**
#skip-changelog

### CODE REVIEW

File: crates/symbolicator/src/services/symcaches.rs [hunk 1]
```diff
@@ -390,6 +390,7 @@ fn write_symcache(
                 handle.uuid,
                 object_handle
             );
+            #[allow(deprecated)]
```
Comment (flub) [inline crates/symbolicator/src/services/symcaches.rs:393]: we deprecated an api we ourselves keep using? 😂

**Review by flub — APPROVED** [review]

File: crates/symbolicator/src/services/symcaches.rs (hunk 1 above)
Comment (Swatinem) [reply crates/symbolicator/src/services/symcaches.rs:393]: :-D it was a bit overeager, I agree. We should migrate to symcache transformers. But that requires unsafe SelfCell code for now. So creating an `OwnedBcSymbolMap` type or something inside symbolic that is `'static` will solve that.

File: crates/symbolicator/Cargo.toml [hunk 2]
```diff
@@ -22,12 +22,12 @@ glob = "0.3.0"
 hostname = "0.3.1"
 humantime-serde = "1.0.1"
 ipnetwork = "0.18.0"
-jsonwebtoken = "7.2.0"
+jsonwebtoken = "8"
```
Comment (jan-auer) [inline crates/symbolicator/Cargo.toml:25]:
```suggestion
jsonwebtoken = "8.0.1"
```

File: crates/symbolicator/Cargo.toml [hunk 3]
```diff
@@ -11,7 +11,7 @@ anyhow = "1.0.38"
 apple-crash-report-parser = "0.4.2"
 backtrace = "0.3.55"
 base64 = "0.13.0"
-cadence = "0.27.0"
+cadence = "0.28"
```
Comment (jan-auer) [inline crates/symbolicator/Cargo.toml:14]:
```suggestion
cadence = "0.28.0"
```

File: crates/symbolicator/Cargo.toml [hunk 4]
```diff
@@ -22,12 +22,12 @@ glob = "0.3.0"
...
 humantime-serde = "1.0.1"
 ipnetwork = "0.18.0"
-jsonwebtoken = "7.2.0"
+jsonwebtoken = "8"
 lazy_static = "1.4.0"
 lru = "0.7.0"
 num_cpus = "1.13.0"
 minidump = "0.9.4"
-parking_lot = "0.11.1"
+parking_lot = "0.12"
```
Comment (jan-auer) [inline crates/symbolicator/Cargo.toml:30]:
```suggestion
parking_lot = "0.12.0"
```

File: crates/symbolicator/Cargo.toml [hunk 5]
```diff
@@ -50,7 +50,7 @@ tracing = "0.1.29"
 tracing-subscriber = { version = "0.3.6", features = ["tracing-log", "local-time", "env-filter", "json"] }
 tokio = { version = "1.0.2", features = ["rt", "macros", "fs"] }
 tokio-metrics = "0.1.0"
-tokio-util = "0.6"
+tokio-util = { version = "0.7", features = ["io"] }
```
Comment (jan-auer) [inline crates/symbolicator/Cargo.toml:53]:
```suggestion
tokio-util = { version = "0.7.0", features = ["io"] }
```

**Review by jan-auer — COMMENTED** [review]
Note that Dependabot will start taking care of this after #709, we could actually skip this manual update.

If you merge this, I'd prefer if we stick to full version numbers for consistency, which matches most documentation examples and also the behavior of `cargo-edit`.

**Pull request merged**